vizro-core/src/vizro/models/_components/graph.py

```diff
@@ -116,6 +116,14 @@ def _filter_interaction(
 
     @_log_call
     def build(self):
+        clientside_callback(
```

Comment:
So now for the example app with 64 graphs when you change theme there's 64 clientside callbacks that execute rather than 1 serverside callback. Performance looks great judging by the videos, but I'm curious  if you know how they're actually executed on the clientside - is it sequentially or in parallel?

Reply:
Testing performances was exactly the reason why I've added 64 graphs.

Javascript is executed in a single thread so it means that everything is executed sequentially (you can literally see how graphs are executed one by one with a debugger). I was aware of that, so I tested the example with 64 graphs. However, even with 64 graphs, everything happens invisibly fast. 😄 

(There is maybe an option to return the Promise from the clientside_callback, but there is nothing "awaitable" (as far as I know) in the function that could be optimised using the concurrent programming and the famous async/await syntax).